qir/qsharp/NegativeBaseProfiles/ExternalCall/QSharpVersion/Example.qs

```diff
@@ -0,0 +1,16 @@
+namespace LoopCase {
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Measurement;
+
+    @EntryPoint()
+    operation Main(nrIter: Int): Int
```

Comment:
This function doesn't seem to correlate to the QIR that is tested in the QSharpTypeQirsNegative.cpp tests. Is this leftover from an unused test case?

Reply:
Except that `nrIter` and `ret` is not used, I see a pretty good correlation with the function in the test:

```llvm
  %q.i = tail call %Qubit* @__quantum__rt__qubit_allocate()
  tail call void @__quantum__qis__h__body(%Qubit* %q.i)
  %result.i.i = tail call %Result* @__quantum__qis__m__body(%Qubit* %q.i)
  tail call void @__quantum__qis__reset__body(%Qubit* %q.i)
```
and
```q#
        use q = Qubit();
        H(q);
        let _ = MResetZ(q);
```
Am I missing something here?

Reply:
Ah, now that you've renamed it I can see which test case it corresponds to, I didn't notice it before. Thanks!